snippets/javascript/jsdoc.json

```diff
@@ -20,22 +20,22 @@
         "prefix": "/*",
         "body": [
             "/**",
-            " * ${1:Comment}$0",
+            " * ${1:Comment.}$0",
```

Comment:
I don't think adding a dot here is necessary

Reply:
All documentation snippets I've been sending follow the convention of respecting correct orthography rules. 

This also allow the user to use sentence level motions on any string (like vis, 'visual inner sentence').

Reply:
my bad I didn't realize it was a string